packages/admin-ui/src/pages/vpn/components/openvpn/OpenVpnDevicesHome.tsx

```diff
@@ -70,19 +74,26 @@ export default function DevicesHome() {
   }
 
   // Input errors
-
   const errors: string[] = [];
   if (input.length > maxIdLength)
     errors.push(`Device name must be shorter than {maxIdLength} characters`);
 
+  // Helper UI in case the OpenVpn client is bad
```

Comment:
Use better comment:
If the OpenVPN package (known as vpn) is not installed, invite the user to install it